snappy/security.go

```diff
@@ -190,15 +191,18 @@ func findWhitespacePrefix(t string, s string) string {
 	return p
 }
 
-func getSecurityProfile(m *packageYaml, appName, baseDir string) (string, error) {
+func getSecurityProfile(m *packageYaml, appName string, baseDir installed.Path) (string, error) {
```

Comment:
making securityProfile a mehtod of installed.Path would be ... weird? ok? circulardependymaking?